### ISSUE TITLE
[Components and pattern standards] Design components and patterns in use are outdated.  (04.08.1)

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#63671]
#### VFS team name
Debt Resolution

#### VFS product name
Financial Status Report (FSR)

#### VFS feature name
Streamlined Waiver

#### Point of Contact/Reviewers
Allison Christman - @allison0034 - Design

*For more information on how to interpret this ticket, please refer to the [Anatomy of a Staging Review issue ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/Anatomy-of-a-Staging-Review-Issue-ticket.2060320997.html) guidance on Platform Website.

---

### Platform Issue
Design components and patterns in use are outdated. 

### Issue Details
There is a wizard at the beginning of the form. I know this was talked about and changing the wizard is going to be implemented at a later date. I am adding this ticket for tracking purposes.

### VA.gov Experience Standard
[Category Number 04, Issue Number 08](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/VA.gov-experience-standards.1683980311.html)

### Other References
WCAG SC 3.2.4 AA

### Platform Recommendation
Go to one thing per page.

---

### VFS Guidance

- Close the ticket when the issue has been resolved or validated by your Product Owner
- If your team has additional questions or needs Platform help validating the issue, please comment on the ticket
- Some feedback provided may be out of scope for your iteration of the product, however, Platform's OCTO leadership has stated that all identified issues need to be documented and it is still your responsibility to resolve the issue.
- If you do not believe that this Staging Review issue ticket is the responsibility of your team, comment below providing an explanation and who you believe is responsible. Please tag the Point of Contact/Reviewers. Governance team will research and will follow up.

**Comment by HeatherRienks** [comment]
FYI- We are working on the Wizard Work in this Epic #48133 and we will keep this ticket and our team ticket #64427 open until all Wizard work is completed!

**Comment by HeatherRienks** [comment]
Closing previous ticket (#64427) as we were awaiting CAIA feedback; @josephrlee created a new ticket #74782 to capture this work.

**Comment by josephrlee** [comment]
Marking this closed as we will be deprecating the wizard and moving it outside of the FSR.

The wizard will be changed to a complete-a-subtask pattern with this epic: https://github.com/department-of-veterans-affairs/va.gov-team/issues/72709 and should be picked back up when the new contract takes effect.

cc: @Tom-Davis @dhill802 @mgleo @denisecoveyduc @charlottecesana
